transform/mattermost-analytics/models/staging/hacktoberboard_prod/_hacktoberboard_prod__models.yml

```diff
@@ -0,0 +1,24 @@
+version: 2
+
+models:
+  - name: stg_hacktoberboard_prod__tracks
+    description: |
+      Reconstructed `tracks` table using custom properties expected to be in the events.
+
+    columns:
+      - name: event_id
+        description: The event's id.
+      - name: event_name
+        description: The name of the event.
+      - name: event_table
+        description: The name of the event table.
+      - name: category
+        description: The event's category.
+      - name: type
```

Comment:
is this `event_type`?

Reply:
Nice catch. I'll do a second round of check for following naming as soon as we merge the PRs.